lib/constants.go

```diff
@@ -645,7 +665,10 @@ var RegtestForkHeights = ForkHeights{
 	// For convenience, we set the block height to 1 since the
 	// genesis block was created using the utxo model.
 	BalanceModelBlockHeight:            uint32(1),
-	ProofOfStakeNewTxnTypesBlockHeight: uint32(1),
+	ProofOfStake1StateSetupBlockHeight: uint32(1),
+
+	// FIXME: set to real block height when ready
+	ProofOfStake2ConsensusCutoverBlockHeight: uint32(math.MaxUint32),
```

Comment:
This is just a placeholder for when we actually write logic for PoS consensus mechanisms, correct? I don't see this fork height used in this PR